manifests/l/Libretro/RetroArch/Nightly/64-bit/1.10.3.0/Libretro.RetroArch.Nightly.64-bit.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Libretro.RetroArch.Nightly.64-bit
+PackageVersion: 1.10.3.0
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://buildbot.libretro.com/nightly/windows/x86_64/RetroArch-Win64-setup.exe
```

Comment:
Wrong URL. Should be https://buildbot.libretro.com/stable/1.10.3/windows/x86_64/RetroArch-Win64-setup.exe
Line 9...